cmd/dockerregistry/main.go

```diff
@@ -16,6 +18,18 @@ import (
 )
 
 func main() {
+	// Start the profiling server, if requested.
+	profilingAddr := os.Getenv("REGISTRY_PROFILING_ADDR")
```

Comment:
>Ah, they only added the port. Well, let's be consistent.

Consistent in what respect? Binding to loopback is of no use if I can't connect to the profiler via a service (did we decide that the problem was the loopback binding or my networking setup)?